Jenkinsfiles/run-job-pipeline.groovy

```diff
@@ -41,48 +41,46 @@ stage("NI: Run ${JOB}") {
                 checkout scm
                 // use json + param files in k8s folder to build/replace job
                 dir("${K8S_PATH}") {
-                    replace_job = sh (
-                        script: """ls""",
-                            returnStdout: true
-                    ).trim()
-                    echo replace_job
-                    // below is commented out code for data loader: use this and newer docs in business-create k8s folder for building the job
-                    // try {
-                    //     replace_job = sh (
-                    //         script: """oc replace jobs/data-loader""",
-                    //             returnStdout: true).trim()
-                    //     echo replace_job
-                    // } catch (Exception e) {
-                           // job doesn't exist yet (hopefully)
-                    //     echo "${e.getMessage()}"
-                    //     data_load_output = sh (
-                    //     script: """oc process -f data-loader.yml -p ENV_TAG=test | oc create -f -""",
-                    //         returnStdout: true).trim()
-                    // }
+                    try {
+                        delete_job = sh (
+                            script: """
+                            oc project ${NAMESPACE}-${TAG_NAME}; \
```

Comment:
Where does this shell run actually?
And I gather it has the oc tools installed already.

Reply:
yes. It's just running within the Jenkins pod that spins up when the pipeline is run